models/app.model.js

```diff
@@ -17,17 +17,34 @@ exports.readTopics = () => {
   })
 }
 
-exports.selectArticles = (sort_by = 'created_at', order = 'desc') => {
+exports.selectArticles = (sort_by = 'created_at', order = 'desc', topic) => {
   const validSortByColumns = ['created_at', 'votes', 'author', 'title', 'article_id', 'comment_count'];
+  const validTopics = ['mitch', 'cats', 'paper'];
```

Comment:
A dynamic way of generating this validTopics array would be good. What if more topics get added to the topics table? This list would have to be updated accordingly every time